load.py

```diff
@@ -81,6 +81,7 @@ def journal_entry(cmdr, system, station, entry, state):
 
     global CONFIG
     paths = [CONFIG.server.get().rstrip("/"),
+	         'api',
```

Comment:
eeek...i thought i did...at first i committed with a tab (yucky) which i thought i fixed in the second commit...

it now lines up (with spaces) with the lines below which i never touched, i'm not sure what else i should do?